spring-cloud-contract-tools/spring-cloud-contract-gradle-plugin/src/test/groovy/org/springframework/cloud/contract/verifier/plugin/ContractVerifierIntegrationSpec.groovy

```diff
@@ -54,13 +54,15 @@ abstract class ContractVerifierIntegrationSpec extends Specification {
 		String gradlePluginSysProp = System.getProperty("contract-gradle-plugin-libs-dir")
 		String gradlePluginLibsDir = (gradlePluginSysProp ?: new File("build/").absolutePath.toString()).replace('\\', '\\\\')
 
-		buildFile.write """
-			buildscript {
-				dependencies {
-					classpath fileTree(dir: '$gradlePluginLibsDir', include: '*.jar')
+		initFile.write """
```

Comment:
Switched to using an initscript here as multiple buildscript sections aren't allowed in the Kotlin DSL. They're technically advised against with the Groovy DSL as well.